CHANGELOG.md

```diff
@@ -3,10 +3,17 @@ Changelog
 
 ## TBD
 
+### Enhancements
+
+* Allows changing the grouping hash when using the logger methods and
```

Comment:
Looks like this sentence got cut off. How does this sound?

```suggestion
* Allows changing the grouping hash when using `BugsnagHandler` via the logger methods' `extra` keyword argument
```

Reply:
Sounds good.